superdesk/media/renditions.py

```diff
@@ -179,7 +181,7 @@ def _resize_image(content, size, format='png', keepProportions=True):
     assert isinstance(size, tuple)
     img = Image.open(content)
     width, height = img.size
-    new_width, new_height = [to_int(x) for x in size]
+    new_width, new_height = [int(x) for x in size]
```

Comment:
I've actually misread, it's size and not img.size, so I've restored the orginal code for this part